bindings/cpp/include/android.h

```diff
@@ -1,3 +1,4 @@
+#ifdef __ANDROID__
```

Comment:
How does it work on compiling (How to use it)?

Reply:
android.h, android.cc is included in play-cpp-sdk.
with CMake, those files are automatically included in compile.

this is to prevent compile errors for non-android targets